### PR TITLE
docs: consistently use active-active failover expression

### PR DESCRIPTION
**Pull request by tisonkun**
Per https://github.com/GreptimeTeam/website/pull/618#pullrequestreview-2193547237

### CODE REVIEW

**Review by sunng87 — APPROVED** [review]
LGTM

File: docs/nightly/en/user-guide/operations/disaster-recovery/dr-solution-based-on-active-active-failover.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# DR solution based on active-active failover
```
Comment (nicecui) [inline docs/nightly/en/user-guide/operations/disaster-recovery/dr-solution-based-on-active-active-failover.md:1]:
```suggestion
# DR Solution Based on Active-Active Failover
```

File: docs/nightly/en/user-guide/operations/disaster-recovery/dr-solution-based-on-cross-region-deployment-in-single-cluster.md [hunk 2]
```diff
@@ -1 +1 @@
-# DR solution based on cross-region deployment in a single cluster
\ No newline at end of file
+# DR solution based on cross-region deployment in a single cluster
```
Comment (nicecui) [inline docs/nightly/en/user-guide/operations/disaster-recovery/dr-solution-based-on-cross-region-deployment-in-single-cluster.md:1]:
```suggestion
# DR Solution Based on Cross-Region Deployment in a Single Cluster
```

File: docs/nightly/en/user-guide/operations/disaster-recovery/overview.md [hunk 3]
```diff
@@ -67,21 +67,21 @@ Write the WAL to the Kafka cluster and store the data in object storage, so the
...
 
 :::tip NOTE
 
-**Dual Active-Standby is only available in GreptimeDB Enterprise.**
+**Active-Active Failover  is only available in GreptimeDB Enterprise.**
 
 :::
 
-For more information about this solution, see [DR solution based on Dual Active-Standby](./dr-solution-based-on-dual-active-standby.md).
+For more information about this solution, see [DR solution based on active-active failover](./dr-solution-based-on-active-active-failover.md).
```
Comment (nicecui) [inline docs/nightly/en/user-guide/operations/disaster-recovery/overview.md:84]:
```suggestion
For more information about this solution, see [DR solution based on Active-Active Failover](./dr-solution-based-on-active-active-failover.md).
```

File: docs/nightly/en/user-guide/operations/disaster-recovery/overview.md [hunk 4]
```diff
@@ -111,7 +111,7 @@ By comparing these DR solutions, you can decide on the final option based on the
 |     DR solution | Error Tolerance Objective |  RPO | RTO | TCO | Scenarios | Remote WAL & Object Storage | Notes |
 | ------------- | ------------------------- | ----- | ----- | ----- | ---------------- | --------- | --------|
 |  DR solution for Standalone| Single-Region | Backup Interval | Minute or Hour level | Low | Low requirements for availability and reliability in small scenarios |  Optional | |
-|  DR solution based on Dual Active-Standby| Cross-Region | 0 | Minute level | Low | High requirements for availability and reliability in small-to-medium scenarios |  Optional | Commercial feature |
+|  DR solution based on active-active failover | Cross-Region | 0 | Minute level | Low | High requirements for availability and reliability in small-to-medium scenarios |  Optional | Commercial feature |
```
Comment (nicecui) [inline docs/nightly/en/user-guide/operations/disaster-recovery/overview.md:114]:
```suggestion
|  DR solution based on Active-Active Failover | Cross-Region | 0 | Minute level | Low | High requirements for availability and reliability in small-to-medium scenarios |  Optional | Commercial feature |
```

File: docs/v0.9/en/user-guide/operations/disaster-recovery/dr-solution-based-on-active-active-failover.md [hunk 5]
```diff
@@ -0,0 +1 @@
+# DR solution based on Active-Active Failover
```
Comment (nicecui) [inline docs/v0.9/en/user-guide/operations/disaster-recovery/dr-solution-based-on-active-active-failover.md:1]:
```suggestion
# DR Solution Based on Active-Active Failover
```

File: docs/v0.9/en/user-guide/operations/disaster-recovery/overview.md [hunk 6]
```diff
@@ -67,21 +67,21 @@ Write the WAL to the Kafka cluster and store the data in object storage, so the
...
 
 :::tip NOTE
 
-**Dual Active-Standby is only available in GreptimeDB Enterprise.**
+**Active-Active Failover  is only available in GreptimeDB Enterprise.**
 
 :::
 
-For more information about this solution, see [DR solution based on Dual Active-Standby](./dr-solution-based-on-dual-active-standby.md).
+For more information about this solution, see [DR solution based on active-active failover](./dr-solution-based-on-active-active-failover.md).
```
Comment (nicecui) [inline docs/v0.9/en/user-guide/operations/disaster-recovery/overview.md:84]:
```suggestion
For more information about this solution, see [DR solution based on Active-Active Failover](./dr-solution-based-on-active-active-failover.md).
```

File: docs/nightly/en/user-guide/operations/disaster-recovery/dr-solution-based-on-cross-region-deployment-in-single-cluster.md (hunk 2 above)
Comment (nicecui) [reply docs/nightly/en/user-guide/operations/disaster-recovery/dr-solution-based-on-cross-region-deployment-in-single-cluster.md:1]: This document is missing in v0.9

Comment (tisonkun) [reply docs/nightly/en/user-guide/operations/disaster-recovery/dr-solution-based-on-cross-region-deployment-in-single-cluster.md:1]: image

It exists. I think that I touch add one new line. We can fix it altogether anyway.

**Review by nicecui — APPROVED** [review]

**Pull request merged**